### DISCUSSION TITLE
Cannot find Visual Query Builder in DBeaver on Linux Mint XFCE.

### DISCUSSION THREAD

**Discussion by ScarletMcLearn** [dbeaver/dbeaver#23207] [Q&A] [answered]
Cannot find Visual Query Builder in DBeaver on Linux Mint XFCE.

1. Right click on DB
2. Click on 'SQL Editor > Create new SQL Script'
3. Open Visual Query Builder from Right side panel
4. There is no right side panel

![Screenshot](https://github.com/dbeaver/cloudbeaver/assets/17823921/e0b44474-87d4-4210-9b22-19c11d172899)

**Comment by LonwoLonwo** [comment]
Hello @ScarletMcLearn 

What exactly version of DBeaver do you have? DBeaver CE? EE? UE? LE?

**Comment by ScarletMcLearn** [comment]
DBeaver CE.

**Comment by Destrolaric** [comment, marked as answer]
@ScarletMcLearn Visual Query Builder is part of PRO editions. It is not present in the CE.
